packages/api/db/migration/20220804164229_estimated_seeds_unit_add_tonnes.js

```diff
@@ -0,0 +1,17 @@
+exports.up = function (knex) {
+  return knex.schema.raw(`
+        ALTER TABLE planting_management_plan
+        DROP CONSTRAINT planting_management_plan_estimated_seeds_unit_check,
+        ADD CONSTRAINT planting_management_plan_estimated_seeds_unit_check
+        CHECK (estimated_seeds_unit = ANY (ARRAY['g'::text, 'kg'::text, 'mt'::text, 'oz'::text, 'lb'::text, 't'::text]));
+    `);
+};
+
+exports.down = function (knex) {
+  return knex.schema.raw(`
+        ALTER TABLE planting_management_plan
+        DROP CONSTRAINT planting_management_plan_estimated_seeds_unit_check,
+        ADD CONSTRAINT planting_management_plan_estimated_seeds_unit_check
+        CHECK (estimated_seeds_unit = ANY (ARRAY['g'::text, 'kg'::text, 'oz'::text, 'lb'::text]));
```

Comment:
This will fail if there is at least one row in the table that is using 'mt' - can you update it to maybe switch all of those to be 't' before adding the constraint?

Reply:
I think we would want it to fail if there is at least one row in the database that has `mt` or `t` right? Since we wouldn't want to invalidate or change the existing data. We don't usually run these down functions from what I understand.

Reply:
@fsayyid I don't think this would really be invalidating the data - wouldn't this just be updating the default display unit because everything in the database is stored in kg?